lib/book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_reader :title, :author, :publication_year
+  def initialize(book_info)
+    @title = book_info[:title]
+    @author = book_info[:author_first_name] + " " + book_info[:author_last_name]
+    @publication_year = book_info[:publication_date].split[-1]
```

Comment:
It might be better to set this year up to only grab the last 4 characters of the string - `book_info[:publication_date][-4..-1]` 

This holds more closely to the spec.